src/toniarts/openkeeper/world/creature/steering/AbstractCreatureSteeringControl.java

```diff
@@ -37,23 +38,27 @@
 
     protected final Creature creature;
     protected SteeringBehavior<Vector2> steeringBehavior;
-    private static final SteeringAcceleration<Vector2> steeringOutput = new SteeringAcceleration<>(new Vector2());
+    private final SteeringAcceleration<Vector2> steeringOutput = new SteeringAcceleration<>(new Vector2());
     private final Vector2 position = new Vector2();
     private float orientation = 0;
     private final Vector2 linearVelocity = new Vector2();
     private float angularVelocity;
     private boolean tagged;
-    private boolean independentFacing = false;
-    private float maxLinearSpeed = 1;
-    private float maxLinearAcceleration = 2;
-    private float maxAngularSpeed = 0.1f;
-    private float maxAngularAcceleration = 0.1f;
+
+    private float maxLinearSpeed = 1f;
+    private float maxLinearAcceleration = 8f;
+    private float maxAngularSpeed = FastMath.RAD_TO_DEG * FastMath.QUARTER_PI / 2;
+    private float maxAngularAcceleration = FastMath.RAD_TO_DEG * FastMath.QUARTER_PI;
+    private float speedThreshold = 0.1f;
     private volatile boolean applySteering = false;
+    // FIXME: dirty hack. Roll back changes and see what happens :-(
+    private volatile boolean stopMoving = false;
 
     public AbstractCreatureSteeringControl(Creature creature) {
         this.creature = creature;
 
         maxLinearSpeed = creature.getSpeed();
+        maxLinearSpeed = maxLinearSpeed * 10;

```

Comment:
`maxLinearAcceleration = maxLinearSpeed * 10;`

but that fix not fixed all bugs :(